src/discordcr/mappings/rest.cr

```diff
@@ -15,5 +15,27 @@ module Discord
         pruned: UInt32
       )
     end
+
+    # A response to the Modify Webhook REST API call.
+    class ModifyWebhookPayload
+      JSON.mapping(name: String?, avatar: String?, channel_id: UInt64?)
+
+      def initialize(@name, @avatar, @channel_id)
+      end
+    end
+
+    class ExecuteWebhookPayload
```

Comment:
Is there any point to having a particular mapping class? It only ever seems to be used once and is never exposed to the user.

Reply:
How should it be changed?